### PR TITLE
Fix sdk link to jira dc

### PR DESCRIPTION
**Pull request by JasBeilin**
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8719

Added regex to catch issues in the new Jira instance.

### CODE REVIEW

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 1]
```diff
@@ -1,42 +1,40 @@
 import argparse
 import re
 import sys
-
 import requests
 import urllib3
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:6]:
```suggestion
from demisto_sdk.commands.common.logger import logger

```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 2]
```diff
@@ -46,25 +44,19 @@ def find_fixed_issue_in_body(body_text, is_merged):
...
+    fixed_issue = [{"link": link, "id": issue_id, "action": 'fixes'} for link, _, issue_id in fixed_jira_issues]
     related_issue = []
 
     if not is_merged:
-        logger.info("Not merging, getting related issues.")
-        related_issue = [
-            {"link": link, "id": issue_id, "action": "relates"}
-            for link, issue_id in related_jira_issue
-        ]
+        print("Not merging, getting related issues.")  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:58]:
```suggestion
        logger.info("Not merging, getting related issues.")
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 3]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
     pr_num = options.pr_num
     username = options.username
     password = options.password
-    instance_url = options.url
+    gold_server_url = options.url
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:72]:
```suggestion
    instance_url = options.url
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 4]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
...
+    instance_url = f"{gold_server_url}/instance/execute/{GENERIC_WEBHOOK_NAME}"
 
-    logger.info(f"Detected Pr: {pr_title=}, {pr_link=}, {pr_body=}")
+    print(f"Detected Pr: {pr_title=}, {pr_link=}, {pr_body=}")  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:75]:
```suggestion
    logger.info(f"Detected Pr: {pr_title=}, {pr_link=}, {pr_body=}")
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 5]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
...
-        and "relates:" not in pr_body.lower()
-        and "fixed:" not in pr_body.lower()
-        and "related:" not in pr_body.lower()
-    ):
-        logger.info("Did not detect Jira linking pattern.")
+    if ("fixes:" not in pr_body.lower()
+            and "relates:" not in pr_body.lower()
+            and "fixed:" not in pr_body.lower()
+            and "related:" not in pr_body.lower()):
+        print("Did not detect Jira linking pattern.")  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:82]:
```suggestion
        logger.info("Did not detect Jira linking pattern.")
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 6]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
...
+        # PR's for Contribution management which don't have a Jira ticket.
         return
 
     issues_in_pr = find_fixed_issue_in_body(pr_body, is_merged)
 
     if not issues_in_pr:
-        logger.error(
-            "ERROR: No linked issues were found in PR. Make sure you correctly linked issues."
-        )
+        print("ERROR: No linked issues were found in PR. Make sure you correctly linked issues.")  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:92]:
```suggestion
        logger.error("ERROR: No linked issues were found in PR. Make sure you correctly linked issues.")
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 7]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
...
 
         sys.exit(1)
 
-    logger.info(f"found issues in PR: {issues_in_pr}")
+    print(f"found issues in PR: {issues_in_pr}")  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:96]:
```suggestion
    logger.info(f"found issues in PR: {issues_in_pr}")
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 8]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
...
-            else "false",  # whether to close the fixed issue in Jira
+            "closeIssue": "true" if is_merged else "false",  # whether to close the fixed issue in Jira
             "PullRequestLink": pr_link,  # will be used to add to jira issue's fields
             "PullRequestTitle": f"[{pr_title}|{pr_link}]",  # will be used in comment of attaching jira issue.
-            "JiraIssues": issues_in_pr,
+            "JiraIssues": issues_in_pr
         },
     }
-    logger.info(body)
+    print(body)  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:108]:
```suggestion
    logger.info(body)
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 9]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
...
     # post to Content Gold
     res = requests.post(instance_url, json=body, auth=(username, password))
 
     if res.status_code != 200:
-        logger.error(
+        print(  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:113]:
```suggestion
        logger.error(
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 10]
```diff
@@ -77,60 +69,58 @@ def trigger_generic_webhook(options):
...
+            f" Gold has status code of {res.status_code}")
         sys.exit(1)
 
     res_json = res.json()
     if res_json and isinstance(res_json, list):
         res_json_response_data = res.json()[0]
         if res_json_response_data:
             investigation_id = res_json_response_data.get("id")
-            logger.info(f"{investigation_id=}")
+            print(f'{investigation_id=}')  # noqa: T201
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:123]:
```suggestion
            logger.info(f"{investigation_id=}")
```

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 11]
```diff
@@ -69,17 +70,17 @@ def trigger_generic_webhook(options):
     pr_num = options.pr_num
     username = options.username
     password = options.password
-    gold_server_url = options.url
+    instance_url = options.url
     instance_url = f"{gold_server_url}/instance/execute/{GENERIC_WEBHOOK_NAME}"
```
Comment (kobymeir) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:74]: where is the gold_server_url defined?

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py [hunk 12]
```diff
@@ -69,17 +70,17 @@ def trigger_generic_webhook(options):
     pr_num = options.pr_num
     username = options.username
     password = options.password
-    gold_server_url = options.url
+    instance_url = options.url
```
Comment (JasBeilin) [inline Utils/github_workflow_scripts/jira_integration_scripts/link_pr_to_jira_issue.py:73]:
```suggestion
    gold_server_url = options.url
```

**Pull request closed without merging**